src/components/App/App.js

```diff
@@ -76,17 +92,18 @@ const App = () => {
       ? <Message error={error} profilePic={""}/>
       : <>
           <Header profilePic={user.attributes.profile_picture_url}/>
+          <Login isLoggedIn={isLoggedIn} logIn={logIn}/>
```

Comment:
If I wanted to disable login while working on other code, would I just comment this out?

Reply:
I would just change the isLoggedIn property in App to true.

Reply:
cool thank you